src/libraries/pretest.proj

```diff
@@ -40,6 +40,7 @@
     <ItemGroup>
       <ManualRuntimePackNativeFile Include="System.Security.Cryptography.Native.OpenSsl.so" />
       <ManualRuntimePackNativeFile Include="System.Security.Cryptography.Native.Apple.dylib" />
+      <ManualRuntimePackNativeFile Include="LdapProxy.Native.dylib" />
```

Comment:
`System.DirectoryServices.*` are independent nuget packages. They are not part of the runtime package.

This will need packagaing changes similar to how System.IO.Ports.Native is packaged. Once that is done, this change should not be needed.